jdaviz/app.py

```diff
@@ -1283,6 +1283,8 @@ def _create_viewer_item(self, viewer, vid=None, name=None, reference=None):
             n = self._next_viewer_num(pfx)
             vid = f"{pfx}-{n}"
 
+        viewer.LABEL = vid  # For reverse look-up
```

Comment:
This is an existing attribute that currently says "Override this" without this patch. Might as well use it?